PHPgettext/streams.php

```diff
@@ -49,7 +49,7 @@ class StringReader {
   var $_pos;
   var $_str;
 
-  function __construct($str='') {
+  function StringReader($str='') {
```

Comment:
@borisson @mlutfy the changes in this file should be reverted (php7 and more so later versions of php5 prefer the __construct style for class constructors